src/arrayMethodSort.js

```diff
@@ -5,7 +5,38 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (typeof compareFunction !== 'function') {
+      compareFunction = (a, b) => {
```

Comment:
Assignment to function parameter 'compareFunction' is an error, don't do like so